1_space_odyssey/js/trond-9000/app.js

```diff
@@ -0,0 +1,30 @@
+//1
+
+var isAlive = true;
+var name = "Trond-9000";
+var hasJetPack = true;
+var suitColor = "white";
+var eyes = true;
+var eyeColor = "blue";
+var hairColor ="dark blonde";
+var legs = 2;
+var arms = 2;
+var hands = 2; 
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = true;
+var hasWeapon = true;
+var isInGoodMood = true;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+//2
+
+if (hasJetPack === true) {
+    console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+}
+else if (hasJetPack === false) {
```

Comment:
if you are only checking one condition or one boolean value, if...else is good enough.

```JavaScript
if (someThing) {
  ...
} else {
 ...
}
```

if you are checking more than one condition then you can use if...else if...else if, and then if there are too many use a switch statement.

![https://media.giphy.com/media/3o85xqzprB9DhZmVk4/giphy.gif](https://media.giphy.com/media/3o85xqzprB9DhZmVk4/giphy.gif)